tests/bluetooth/controller/ctrl_feature_exchange/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# some of the control procedures in the BT LL depend on
+# the following configs been set
+config SOC_COMPATIBLE_NRF
+       bool
```

Comment:
Didn't know this, but makes sense. A cleanup of other Kconfig files may be required then, for example in tests/bluetooth/df/* (where this was copied from)

Reply:
specifying the type has been removed from all other places as well

Reply:
> A cleanup of other Kconfig files may be required then, for example in tests/bluetooth/df/* (where this was copied from)

Yes, there's a bit of whack-a-mole here, because developers naturally copies files / code snippes around and adjust.

Thanks for cleaning up.